### PR TITLE
Refactoring ClientPoliciesTest

### LINKED ISSUE #14795: Refactoring ClientPoliciesTest
### Description

Current `ClientPoliciesTest` is too much big so that it might not be easy for us to maintenance.
Therefore, it is needed to refactor `ClientPoliciesTest` to make it easy to maintain.

For, example, separating tests for checking newly added client policy events.

This task is a follow-up task of https://github.com/keycloak/keycloak/issues/12065 .

### PR DESCRIPTION
**Pull request by MuhammadZakwan**
Closes #14795

### CODE REVIEW

**Review by tnorimat — DISMISSED** [review]
I have confirmed that all test methods of the original `ClientPoliciesTest` were migrated. LGTM.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/policies/ClientPoliciesAdminTest.java [hunk 1]
```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
```
Comment (sguilhen) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/policies/ClientPoliciesAdminTest.java:2]: Nitpick: year. Same for all new files in the PR.

**Review by sguilhen — COMMENTED** [review]
Overall looks good to me - only one minor note about year in new files' headers.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/policies/ClientPoliciesAdminTest.java (hunk 1 above)
Comment (mposolda) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/policies/ClientPoliciesAdminTest.java:2]: +1

**Review by mposolda — CHANGES_REQUESTED** [review]
@MuhammadZakwan @tnorimat Thanks for the change and for the review of this PR!

I have a small concern: Is it possible to revert your changes in the files, where are no changes besides imports? There are no changes needed in the files as the "imports" were already present, but in your PR, you are moving them to the different lines, which adds some unnecessary noise... It applies to the files `CIBATest`, `FAPI1Test`, `FAPICIBATest`, `ParTest`, `ClientPoliciesFeatureTest`, `ClientPoliciesImportExportTest`, `ClientPoliciesLoadUpdateTest`.

Also I suggest to update your IDE settings to avoid updating "imports" if there is no reason for it (EG. There is reason when you're adding or removing "imports", but not needed to change the order of "import" lines).

Also +1 for @sguilhen comment about the year.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/policies/ClientPoliciesAdminTest.java (hunk 1 above)
Comment (tnorimat) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/policies/ClientPoliciesAdminTest.java:2]: +1

**Review by tnorimat — DISMISSED** [review]
LGTM

**Comment by tnorimat** [comment]
@mposolda I have re-reviewed and approved it. Could you check it?

**Review by mposolda — DISMISSED** [review]
@tnorimat @MuhammadZakwan Thanks for the updates and re-review.

It looks good. I will try to re-run the CI to doublecheck we are good as it was not fully executed with this PR for some reason...

**Comment by mposolda** [comment]
@MuhammadZakwan Could you please try to rebase your PR to latest Keycloak main? The main testsuite was not executed with your PR and I am not sure why. So hopefully rebase will help...

**Review by mposolda — APPROVED** [review]
@MuhammadZakwan Thanks!

**Pull request merged**